### PR TITLE
Add FPort description and setup pages

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-FPort-receivers.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+===============
+FPort Receivers
+===============
+
+FPort protocol combines SBus RC control information being sent to the autopilot with bi-directional telemetry information to/from the autopilot on a single wire high baud rate bus.
+
+Many FRSky X series and R series receivers have this capability, either natively or with firmware upgrade.
+
+Connection to the autopilot and configuration varies from autopilot to autopilot since their UART capabilities vary. Most F4 cpu based autopilots do not controllable inverters in front of the UART TX and RX, and require either an external bi-directional inverter, as SPort connections do (see :ref:`frsky_cables` ), or the use of the "uninverted FPort pad/pin" some receivers provide since the FPort protocol has opposite signaling levels than normal UART operation.
```
Comment (tridge) [inline common/source/docs/common-FPort-receivers.rst:11]: "do not controllable" should be "do not have controllable"

File: common/source/docs/common-FPort-receivers.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+Most F7 and H7 autopilot, on the other hand, have internal inverters,and can be directly connected from the receivers FPort to their UARTs TX input line, as shown below:
```
Comment (tridge) [inline common/source/docs/common-FPort-receivers.rst:12]: missing space, plural autopilot

File: common/source/docs/common-FPort-receivers.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+
+.. note:: some boards can have the FPort connected to the UARTs RX pin instead of TX pin as shown above. See below:
+
+Configuration
+=============
+
+Configuration depends on the autopilot and connection method.
+
+.. note:: Any configuration change to a UART requires a re-boot to take effect.Reminder: UART numbers do not necessarily map to SERIALx numbers. Check the :ref:`board's description<common-autopilots>`
```
Comment (tridge) [inline common/source/docs/common-FPort-receivers.rst:34]: missing space

File: common/source/docs/common-rssi-received-signal-strength-indication.rst [hunk 4]
```diff
@@ -53,7 +53,7 @@ There are four basic options for feeding RSSI to your autopilot:
   :ref:`RSSI_CHAN_LOW<RSSI_CHAN_LOW>`  PWM value the radio receiver will output when the signal is the weakest.
 
 
- :ref:`RSSI_TYPE<RSSI_TYPE>` = 3  digital receiver protocols' signal quality or dropped packts information (SUMD / ST24)
+ :ref:`RSSI_TYPE<RSSI_TYPE>` = 3  digital receiver protocols' signal quality or dropped packts information (SUMD / ST24 /FPort)
```
Comment (tridge) [inline common/source/docs/common-rssi-received-signal-strength-indication.rst:56]: packts typo

File: common/source/docs/common-FPort-receivers.rst [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+F4 based autopilots using an "inverted FPort" output from the receiver can connect it to any  UART's TX pin with the configuration:
+
+-   ``SERIALx_PROTOCOL`` =23
+-   ``SERIALx_OPTIONS`` = 4 (Half Duplex) **untested**
+-   ``RSSI_TYPE`` =3
+
+    OR to the RX pin with the following configuration (since some autopilot boards have only the RX pinned out for some UARTS for ESC telemetry):
+
+-   ``SERIALx_PROTOCOL`` =23
+-   ``SERIALx_OPTIONS`` = 12 (SWAP TX and RX, Half Duplex) **untested**
```
Comment (tridge) [inline common/source/docs/common-FPort-receivers.rst:52]: swap option doesn't work on F4

File: common/source/docs/common-FPort-receivers.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:11]: fixed

File: common/source/docs/common-FPort-receivers.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:12]: fixed

File: common/source/docs/common-FPort-receivers.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:34]: fixed

File: common/source/docs/common-rssi-received-signal-strength-indication.rst (hunk 4 above)
Comment (Hwurzburg) [reply common/source/docs/common-rssi-received-signal-strength-indication.rst:56]: fixed

File: common/source/docs/common-FPort-receivers.rst (hunk 5 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:52]: fixed

File: common/source/docs/common-FPort-receivers.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:12]: fixed

File: common/source/docs/common-rssi-received-signal-strength-indication.rst (hunk 4 above)
Comment (Hwurzburg) [reply common/source/docs/common-rssi-received-signal-strength-indication.rst:56]: fixed

File: common/source/docs/common-FPort-receivers.rst (hunk 5 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:52]: fixed and added note

File: common/source/docs/common-FPort-receivers.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:11]: fixed

File: common/source/docs/common-FPort-receivers.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-FPort-receivers.rst:34]: fixed

**Pull request merged**